test/jdk/java/awt/Focus/UnaccessibleChoice/AccessibleChoiceTest.java

```diff
@@ -89,15 +95,16 @@ public void start () {
         } catch (Exception ex) {
             throw new RuntimeException("Can't create robot");
         }
+        robot.waitForIdle();
         robot.delay(2000);
```

Comment:
Is such a large delay required here since `autoDelay` and `waitForIdle` are also being called?

Reply:
I believe the proper amount is a delay of 1000. I can reduce this, but I thought it might be best to leave as much unchanged as possible. It does make sense to reduce it to 1000 if possible though.